spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1234,4 +1234,31 @@ describe Crystal::Formatter do
     # 3 + 4
     # ```
     CODE
+
+  # Keep trailing spaces in macros.
```

Comment:
Editors are usually configured to trim white spaces at the end.
This specs should be written different in order to avoid loosing those spaces.